### PR TITLE
Iteration two

### PR DESCRIPTION
**Pull request by zliibbe**
## Purpose:
Completes Iteration 2 of Travel Tracker. 

- [x] New feature (non-breaking change which adds functionality)
- [ ] Improves an existing feature
- [ ] Fixes a bug
      - Bug fixed:
- [ ] Refactor
      - Code refactored:
- [ ] Documentation update

## Description: What does this PR do?
This branch allows a traveler to make a trip request using the form below user info. After making these selections and clicking submit, the new trip posts to the server and display on the DOM with a status of pending. A message briefly displays the success of posting new Trip to the server.  See new form here:
image
User is able to scroll down to trip to see status and cost. 

closes issue #2

## Instructions for reviewer:
*Where should the reviewer start?* apiCalls.js, then scripts.js, domUpdates.js, and index.html as needed. 
*How should this be manually tested?*Testing can be done in the browser. I tested with console.log and review of the DOM in the browser.
*Any background context you want to provide?* Styling is lacking IMHO, however, I'm moving forward to pursue functionality over style. 
*What are the relevant tickets?* closes issue #2

## Issues:
All methods are functional and passing required testing. Everything displays appropriately per the Turing Spec. 

## Checklist:
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have used tests to prove my fix is effective or that my feature works
- [x] My changes generate no new warnings
- [x] New and existing unit tests pass locally with my change

## Future Steps:

- Make branch for accessibility check (per Turing's Project Spec)
- Complete Iteration 3 (Login Screen)

### CODE REVIEW

File: src/css/base.scss [hunk 1]
```diff
@@ -27,23 +55,79 @@ h1 {
 .trip {
   position: relative;
   text-align: center;
+  @include setDimensions(40vh, 100%);
+  // border: 2px solid blue;
+  // @include displayFlex(row, space-evenly, nowrap)
+}
```
Comment (Andrew-Musselman) [inline src/css/base.scss:61]: Some commented out code here, and line 72-73

File: src/scripts.js [hunk 2]
```diff
@@ -41,6 +49,36 @@ const getByID = (travelers, id) => {
...
+        travelersData = data[0].travelers;
+        destinationsData = data[1].destinations;
+        tripsData = data[2].trips;
+        user.trips = user.getUsersTrips(tripsData, destinationsData)
+        userDestinations = user.getUsersDestinations(destinationsData)
+    }).then(() =>{
+        updateDOM(user, user.trips, userDestinations)//redisplay new info to DOM
+    })
+    e.target.reset();
+}
```
Comment (Andrew-Musselman) [inline src/scripts.js:80]: This looks good!

**Review by Andrew-Musselman — APPROVED** [review]
Everything looks good, just a few commented out code statements in css that should be cleaned up

**Review by CHaiz15 — APPROVED** [review]
Looks good! Just one comment to look at. Lmk if it doesn't make sense

**Pull request merged**